src/classes/install-sw-refs.ts

```diff
@@ -0,0 +1,92 @@
+import { Software } from "../interfaces/types";
+
+export class InstallSwRefs {
+
+  /**
+   * Get Install Reference for Installation Utils class
+   * 
+   * @param software is the software in question
+   * 
+   * @returns installation reference used in bash to install the software
+   */
+  public static async getInstallRef(installUtil : string, software : string) : Promise<string> {
+
+    switch (software) {
+      case Software.NodeJs:
+        if (installUtil == "brew") {
+          return "node";
+        } else {
+          return "nodejs";
+        }
```

Comment:
could use a shorter syntax on many of these
```suggestion
       return (installUtil === "brew") ? "node" : "nodejs";
```